src/constants.ts

```diff
@@ -1,4 +1,16 @@
-const DEFAULT_CHE_IMAGE = 'quay.io/eclipse/che-server:nightly'
-const DEFAULT_CHE_OPERATOR_IMAGE = 'quay.io/eclipse/che-operator:nightly'
+/*********************************************************************
+ * Copyright (c) 2020 Red Hat, Inc.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ **********************************************************************/
 
-export { DEFAULT_CHE_IMAGE, DEFAULT_CHE_OPERATOR_IMAGE }
+export const DEFAULT_CHE_IMAGE = 'eclipse/che-server:nightly'
+export const DEFAULT_CHE_OPERATOR_IMAGE = 'quay.io/eclipse/che-operator:nightly'
+
+export const CERT_MANAGER_NAMESPACE_NAME = 'cert-manager'
+
+export const CHE_SECRET_NAME = 'che-tls'
```

Comment:
It's tls specific secret, right? Then let's make it clearer with constant name

Reply:
ok